### PR TITLE
Spinel Changes

### PR DESCRIPTION
**Pull request by darconeous**
This pull request contains various minor changes to help with readability.

### CODE REVIEW

**Comment by jhwoodyatt** [comment]
LGTM.

File: doc/spinel-protocol-src/draft-rquattle-spinel-basis.md.in [hunk 1]
```diff
@@ -50,23 +49,13 @@
...
+This document specifies the basis of the Spinel protocol, which facilitates the control and management of IPv6 network interfaces on devices where general purpose application processors offload network functions at their interfaces to network co-processors (NCP) connected by simple communication links like serial data channels. Spinel was initially designed for use with Thread network co-processors, but its basis is general purpose and intended to be easily adapted to other types of IPv6 network interface.
 
 {mainmatter}
 
 # Introduction #
 
-Spinel is designed to enable the interoperation over simple serial connections between general purpose device operating systems (OS) and network co-processors (NCP) for the purpose of controlling and managing their IPv6 network interfaces, achieving the following goals:
+Spinel is host-controller protocol designed to enable interoperation over simple serial connections between general purpose device operating systems (OS) and network co-processors (NCP) for the purpose of controlling and managing their IPv6 network interfaces, achieving the following goals:
```
Comment (jwhui) [inline doc/spinel-protocol-src/draft-rquattle-spinel-basis.md.in]: nit: "is host-controller" -> "is **a** host-controller"?

File: doc/spinel-protocol-src/draft-rquattle-spinel-unified.md.in [hunk 2]
```diff
@@ -51,42 +50,22 @@
...
-this document.
 
 {mainmatter}
 
 # Introduction #
 
-This Network Control Processor (NCP) protocol was designed to enable a
-host device to communicate with and manage a NCP while also achieving
-the following goals:
+Spinel is host-controller protocol designed to enable interoperation over simple serial connections between general purpose device operating systems (OS) and network co-processors (NCP) for the purpose of controlling and managing their IPv6 network interfaces, achieving the following goals:
```
Comment (jwhui) [inline doc/spinel-protocol-src/draft-rquattle-spinel-unified.md.in]: nit: "is host-controller" -> "is **a** host-controller"

File: doc/spinel-protocol-src/spinel-prop-overview.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+*   Single-value properties
+*   Multiple-value (Array) properties
+*   Stream properties
+
+### Single-Value Properties ####
+
+Single-value properties are properties that have a simple representation of a single value. Examples would be:
+
+*   Current radio channel (Represented as a unsigned 8-bit integer)
```
Comment (jwhui) [inline doc/spinel-protocol-src/spinel-prop-overview.md]: nit: "as a unsigned" -> "as **an** unsigned"

File: doc/spinel-protocol-src/spinel-prop-overview.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+*   Network name (Represented as a UTF-8 encoded string)
+*   802\.15.4 PAN ID (Represented as a unsigned 16-bit integer)
```
Comment (jwhui) [inline doc/spinel-protocol-src/spinel-prop-overview.md]: nit: "as a unsigned" -> "as **an** unsigned"

**Comment by darconeous** [comment]
In the future, I think minor grammar errors should be addressed in follow up commits until we get this moved over into a new repository. I would have addressed these issues directly in the GitHub web interface, but I can't pick the email address used in the attribution and that causes googlebot to wig out.

**Pull request merged**